x-pack/plugins/security_solution/public/management/components/console/components/side_panel/side_panel_flex_item.tsx

```diff
@@ -13,7 +13,6 @@ import { useWithSidePanel } from '../../hooks/state_selectors/use_with_side_pane
 
 const StyledEuiFlexItemWhite = styled(EuiFlexItem)`
   background-color: ${({ theme: { eui } }) => eui.euiHeaderBackgroundColor} !important;
-  border-radius: ${({ theme: { eui } }) => eui.euiSizeXS};
```

Comment:
This would make the top and bottom right and corners of the responder squared. I would suggest you do this instead,

```
border-radius: ${({ theme: { eui } }) => `0 ${eui.euiSizeXS} ${eui.euiSizeXS} 0`};
```